src/Handler/ExceptionHandler.php

```diff
@@ -101,13 +101,16 @@ public function __invoke(
                 $response = $e->withResponse($response);
             }
 
-            $handler = $this->handler($type);
-            $this->whoops->pushHandler($handler);
+            if ($this->preferences->displayDebug( )) {
```

Comment:
why the space between the parens in `displayDebug(  )`?

Reply:
Just habit from a coding standard I used for a long time.  Fixed to match project style.